design-documents/graph-ql/coverage/CartPrices.graphqls

```diff
@@ -0,0 +1,57 @@
+interface CartItemInterface {
+    prices: CartItemTotals
+}
+
+type Cart {
+    prices: CartTotals
+    addresses: [CartAddress]
+    # additional fields will be added later
+}
+
+type CartAddress {
+    pices: CartAddressTotals
```

Comment:
Is this the CartAddressPrices? Also, note the small typo, the field name should be `prices`

Reply:
Why plural?

Reply:
I think plural makes sense because there are multiple price types stored in this field. 
Do you strongly believe it should be singular in this case?

Reply:
Not really, but there's this small inconsistency where you have a plural to designate lists but here is a single object. Why don't name it totals?